x-pack/filebeat/input/httpjson/internal/v2/response.go

```diff
@@ -175,10 +175,10 @@ func (resp *response) templateValues() common.MapStr {
 		return common.MapStr{}
 	}
 	return common.MapStr{
-		"header":    resp.header.Clone(),
-		"page":      resp.page,
-		"url.value": resp.url.String(),
-		"params":    resp.url.Query(),
-		"body":      resp.body,
+		"header":     resp.header.Clone(),
+		"page":       resp.page,
+		"url.value":  resp.url.String(),
+		"url.params": resp.url.Query(),
```

Comment:
Any chance we could run this through a test render to validate the template in the case of any future changes?